build_tools/benchmarks/suites/iree_bytecode_module_template.cmake

```diff
@@ -0,0 +1,15 @@
+# Compile the module "$__MODULE_OUTPUT_PATH"
+iree_bytecode_module(
```

Comment:
This is doing something very similar to Bazel->CMake. I think it may make sense to use a similar approach in the two. Bazel->CMake builds up strings rather than using template files. Part of the reason is that some arguments may be optional. I'm not sure if we're going to have that case here though.

Reply:
I feel that at small scale the template is easier to read (these are basically all the templates). I'll try locally to see if Bazel->CMake's builder method is better.